synnax/pkg/distribution/framer/streamer.go

```diff
@@ -83,6 +83,7 @@ func (l *streamer) Flow(sCtx signal.Context, opts ...confluence.Option) {
 			confluence.Drain(l.iter.responses)
 		}
 
+		// HERE
```

Comment:
This is part of a `Flow` call to the framer streamer. Instead of adding a panic policy here, I will just add a panic policy to the callers of this `Flow` method – currently the only caller is `signals/subscriber`.